SPECS/wget2/wget2.spec

```diff
@@ -0,0 +1,210 @@
+%global as_wget 1
+%global somajor 2
+
+Name:           wget2
```

Comment:
Please make this just wget and remove the old wget.

Reply:
ack